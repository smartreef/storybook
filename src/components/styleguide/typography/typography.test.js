import React from 'react';
import renderer from 'react-test-renderer';
import TypographyBlock from './';

it('match snapshot', () => {
	const component = renderer.create(<TypographyBlock />);
	const tree = component.toJSON();

	expect(tree).toMatchSnapshot();
});
