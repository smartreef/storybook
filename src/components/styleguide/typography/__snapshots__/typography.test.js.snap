// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="mb-6"
>
  <p
    className="text-sm text-gray-600"
  >
    .
    text-base font-normal leading-normal tracking-normal
  </p>
  <p
    className="text-base font-normal leading-normal tracking-normal"
  >
    The quick brown fox jumps over the lazy dog
  </p>
</div>
`;
